hyperspy/_components/expression.py

```diff
@@ -118,6 +118,11 @@ class docstring.
             self.__doc__ = _CLASS_DOC % (
                 name, sympy.latex(_parse_substitutions(expression)))
 
+        for parameter in self._parameter_strings:
```

Comment:
Thanks... I thought I had tried that initially and it didn't work. I must have confused myself somehow. Will fix tomorrow.